pello/__main__.py

```diff
@@ -0,0 +1,7 @@
+from pello import pello_greeting
+
+def main():
+    pello_greeting.greeting()
+
+if __name__ == "__main__":
+    main()
```

Comment:
name is always `__main__` because this module is named `__main__` (the conditional is not helpful here).